### DISCUSSION TITLE
Cannot open / create Office file

### DISCUSSION THREAD

**Discussion by vawaver** [nextcloud/all-in-one#1651] [Questions] [answered]
### Steps to reproduce
1. AIO Updated from 3.2.0 -> 4.0.1
2. Cannot create/load Office documents (files).
3.

### Expected behavior 
If I try to load or create an Office file then this file is opened or created and the Office interface is loaded correctly in my browser.

### Actual behavior 
I try to open or create Office documents via a web browser. The process of opening or editing has started. The process ended with the error " Document loading failed. Failed to load Nextcloud Office - please try again later." after while.

### Host OS 
Ubuntu server 22.0.1
#### Other valuable info 
![aio_loadingOfficeFile](https://user-images.githubusercontent.com/52236863/210239166-f6baa8ae-ba5e-4ea1-99f9-4dd40f08d2fc.png)

**Comment by szaimen** [comment]
Hi, can you follow https://github.com/nextcloud/all-in-one/discussions/1358?

**Reply by calidancer** [reply]
Thanks @szaimen. It worked also for me.

**Comment by vawaver** [comment, marked as answer]
> Hi, can you follow #1358?

Thank you for the link, bro!
,0.0.0.0/0 - to the WOPI field solved the problem.

**Comment by szaimen** [comment]
All right, great! Moving to discussions in order to help others that experience the same.
